doc/classes/TileMap.xml

```diff
@@ -173,6 +173,31 @@
 				[/codeblock]
 			</description>
 		</method>
+		<method name="set_cell_region">
+			<return type="void">
+			</return>
+			<argument index="0" name="start" type="Vector2" default="Vector2( 0, 0 )">
+			</argument>
+			<argument index="1" name="end" type="Vector2" default="Vector2( 0, 0 )">
+			</argument>
+			<argument index="2" name="tile" type="int" default="-1">
+			</argument>
+			<argument index="3" name="flip_x" type="bool" default="false">
+			</argument>
+			<argument index="4" name="flip_y" type="bool" default="false">
+			</argument>
+			<argument index="5" name="transpose" type="bool" default="false">
+			</argument>
+			<argument index="6" name="autotile_coord" type="Vector2" default="Vector2( 0, 0 )">
+			</argument>
+			<description>
+				Sets the tiles in the given region.
```

Comment:
Suggestion to avoid repeating the function name:

Assigns the tile of id `tile` to all cells in the rectangular region defined by `start` and `end`.